Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebExtension.cpp

```diff
@@ -110,11 +112,47 @@ using namespace WebKit;
  *   // ...
  * }
  * ```
+ *
+ * To send user data related to a particular #WebKitWebView, you have also the possibility of using
+ * the #WebKitWebView::initialize-web-extensions signal in combination with the
+ * webkit_web_view_set_web_extensions_initialization_user_data():
+ *
+ * ```c
+ * g_signal_connect(view, "initialize-web-extensions", G_CALLBACK(initialize_web_extensions), this);
+ *
+ * static void
+ * initialize_web_extensions (WebKitWebView* view, gpointer* user_data)
```

Comment:
```suggestion
 * initialize_web_extensions (WebKitWebView *view, gpointer user_data)
```

Note we have only a gpointer, not a `gpointer*`.